app/guides/branching.md

```diff
@@ -8,24 +8,118 @@ tags:
 
 To do this, add some JavaScript to your `routes.js` file.
 
-Here is an example:
+## Branching on radio button answers
+
+Send users to different pages based on their radio button selection:
+
+```js
+router.post('/eligibility-answer', function (req, res) {
+  const data = req.session.data
+  const eligible = data.isEligible
+
+  if (eligible == "Yes") {
+    res.redirect('/next-question')
+  }
+  else {
+    res.redirect('/not-eligible')
+  }
+})
+```
+
+### Multiple options
+
+Handle more than 2 options with `else if`:
+
+```js
+router.post('/appointment-type-answer', function (req, res) {
+  const data = req.session.data
+  const appointmentType = data.appointmentType
+
+  if (appointmentType == "Face to face") {
+    res.redirect('/book-face-to-face')
+  }
+  else if (appointmentType == "Phone call") {
+    res.redirect('/book-phone')
+  }
+  else if (appointmentType == "Video call") {
+    res.redirect('/book-video')
+  }
+  else {
+    // No answer selected, return to question
+    res.redirect('/appointment-type')
+  }
+})
+```
+
+## Branching on checkbox answers
+
+Checkboxes return an array of values. Check if a specific value was selected:
+
+```js
+router.post('/symptoms-answer', function (req, res) {
+  const data = req.session.data
+  const symptoms = data.symptoms
+
+  // Check if 'Fever' was selected
+  if (symptoms && symptoms.includes('Fever')) {
+    res.redirect('/high-risk')
+  }
+  else {
+    res.redirect('/standard')
+  }
+})
+```
+
+### Multiple checkbox conditions
+
+Check for multiple selected values:
 
 ```js
-// Called when answering question about whether NHS number is known
-router.post("/do-you-know-your-nhs-number-answer", (req, res) => {
-  // Make a variable and give it the value from 'nhsNumberKnown'
-  const nhsNumberKnown = req.session.data.nhsNumberKnown
-
-  // Check whether the variable matches a condition
-  if (nhsNumberKnown === "Yes") {
-    // Send user to a page where they'll enter their NHS number
-    res.redirect("/enter-nhs-number")
-  } else if (nhsNumberKnown === "No") {
-    // Send user to a page where they can find their NHS number
-    res.redirect("/find-nhs-number")
-  } else {
-    // Send user back to the question page
-    res.redirect("/do-you-know-your-nhs-number")
+router.post('/symptoms-answer', function (req, res) {
+  const data = req.session.data
+  const symptoms = data.symptoms
+
+  if (
+    symptoms &&
+    symptoms.includes('Chest pain') &&
+    symptoms.includes('Breathlessness')
+  ) {
+    res.redirect('/urgent')
+  }
+  else if (symptoms && symptoms.includes('Fever')) {
+    res.redirect('/high-risk')
+  }
+  else {
+    res.redirect('/standard')
   }
```

Comment:
This whole example seems quite similar to the prior one.

Reply:
Agree, have merged. Also changed the first condition to be an `||` so that the logic makes sense with a 'None of these' option.